### DISCUSSION TITLE
Exporting trpc app router without exposing backend logic

### DISCUSSION THREAD

**Discussion by jericopulvera** [trpc/trpc#2096] [Q&A] [answered]
The use case is to share trpc/server app router for those who want to create a different client app that uses react and installs trpc/client.

**Comment by KATT** [comment, marked as answer]
Check out https://github.com/jlalmes/trpc-openapi if you wanna make an open API.

You can definitely export only the types of the tRPC router, but unsure how much of the internals it would leak
